pkg/endpoints/pipeline_test.go

```diff
@@ -100,6 +100,21 @@ func TestTask(t *testing.T) {
 	if result1.Name != "Task2" {
 		t.Errorf("Task2 is not returned: %s", result1.Name)
 	}
+
+	//get tasks in all namespaces
+	httpReq = dummyHTTPRequest("GET", "http://wwww.dummy.com:8383/v1/namespaces/*/tasks/", nil)
+	req = dummyRestfulRequest(httpReq, "*", "")
+	httpWriter = httptest.NewRecorder()
+	resp = dummyRestfulResponse(httpWriter)
+
+	r.getAllTasks(req, resp)
+
+	result = v1alpha1.TaskList{}
+	json.NewDecoder(httpWriter.Body).Decode(&result)
+
+	if len(result.Items) != 3 {
```

Comment:
Do not use static values, compare against the length of CRD array whose element have all been created